README.md

```diff
@@ -1,13 +1,14 @@
 # Picnic Reactive Support
 
+[![Build Status](https://travis-ci.org/PicnicSupermarket/reactive-support.svg?branch=master)](https://travis-ci.org/PicnicSupermarket/reactive-support)
+
 A collection of Reactive Programming (RxJava and Reactor) utilities forged and
 implemented in the Picnic backend.
 
 ## How to Install
 
-### Gradle
-
 Using [JitPack][jitpack]:
+### Gradle
```

Comment:
- Empty line.
- The GAVs below won't be correct. Suggest we first try with Jitpack by pointing to a commit and iterate before merging this PR.